src/udx.c

```diff
@@ -949,20 +981,22 @@ udx__confirm_packet (udx_packet_t *pkt) {
 
     // rack 7.3
     if (pkt->is_tlp) {
+      stream->tlp_in_flight = true;
```

Comment:
unrelated to stats?

Reply:
yes, it was fixed separately in holepunchto/libudx here: https://github.com/holepunchto/libudx/pull/186